package.json

```diff
@@ -27,24 +27,18 @@
     "project.yaml"
   ],
   "dependencies": {
-    "@babel/preset-typescript": "^7.15.0",
-    "@polkadot/api": "5.1.1",
-    "@polkadot/util": "7.2.1",
-    "@polkadot/util-crypto": "7.2.1",
+    "@polkadot/api": "^8",
     "bignumber.js": "^9.0.2",
-    "bn.js": "^4.11.9",
     "typeorm": "^0.2.34"
   },
-  "author": "raycar5",
+  "author": "Prashant",
```

Comment:
Should this be "Polymath" or something along those lines?

Reply:
I guess this got replaced when I did `subql init`. I'll replace with Polymath or should it be Polymesh?

Reply:
I am not too sure. In the SDK our error messages say please contact the Polymath team, although Polymesh makes more sense to me.